cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/FileStreamAdmin.java

```diff
@@ -325,9 +327,21 @@ public void drop(Id.Stream streamId) throws Exception {
     truncate(streamId);
   }
 
+  private Location getStreamConfigLocation(Id.Stream streamId) throws IOException {
+    return getStreamBaseLocation(streamId).append(CONFIG_FILE_NAME);
+  }
+
+  // Constructs path: /.../<namespace>/streams/<streamName>, as expected by StreamUtils#getStreamIdFromLocation
+  private Location getStreamBaseLocation(Id.Stream streamId) throws IOException {
```

Comment:
Is that okay that we call stream base location either the location at the namespace directory, or the location at the stream directory?